var/spack/repos/builtin/packages/n2p2/package.py

```diff
@@ -94,7 +94,8 @@ def setup_build_tests(self):
         install test subdirectory for use during `spack test run`."""
         self.cache_extra_test_sources(".")
```

Comment:
This needs to be called:

```suggestion
        cache_extra_test_sources(self, ".")
```